### DISCUSSION TITLE
How can I use spate Ray app for WordPress debug?

### DISCUSSION THREAD

**Discussion by mhraihan** [spatie/wordpress-ray#72] [Q&A]
I'm Using `Local` for WordPress plugin development.

I'm trying to configure the `Ray app` for it. I already installed and activated the Ray Desktop app and the `WordPress-ray` plugin for it.

also this one configured as the documentation `define( 'WP_ENVIRONMENT_TYPE', 'local' );`

but it does not work for me.

if `ray("Ray");` is defined in my code, I've got some errors Fatal error: Uncaught Error: Call to undefined function MH_Slider\ray().

How may I configure its proper way and solve the problem?

**Comment by jeffreyvr** [comment]
Looks like a namespace issue seeing as it's tries to look in the `MH_Slider` namespace. Try calling ray like `\ray()` (prepend with a backward slash).

**Reply by mhraihan** [reply]
WordPress-ray plugin does not work for me.
After installing ray globally, I can use it.
thank you
